### ISSUE TITLE
It seems a bit different from WAMP specification

### ISSUE THREAD

**Issue by Kenzku** [nicokaiser/wamp.io#3]
I am not sure if I understand correctly, it seems your server only accept PUBLISH and the format is a bit different from WAMP specification? http://wamp.ws/spec#publish_message

It says: publish (a) message with 'string' as payload [7, "http://example.com/simple", "Hello, world!"]

While your code:
        try {
            var parsed = qs.parse(buffer);
            console.log(parsed);
            topicUri = parsed.topicuri;
            event = JSON.parse(parsed.body);
        } 

It seems you need "topicuri" and "body", but the server does not accept: {"topicuri":"http://example.com/simple","body":"something"} or topicuri=http://example.com/simple&body=something
(based on https://developer.mozilla.org/en-US/docs/JavaScript/Reference/Global_Objects/JSON/parse, it should accept body=something)

While it will accept: topicuri=http://example.com/simple&body={"event":"helleo"}
(and this way is different from the specification, maybe?)

![Screen Shot 2013-03-01 at 16 27 03](https://f.cloud.github.com/assets/1387249/209806/20b3959a-827c-11e2-9d53-e65e8aaf8785.png)

**Comment by nicokaiser** [comment]
You are right, the little server in the example code only supports PUBLISH – while SUBSCRIBE can be done in the client (which is not implemented here, you can use the original Autobahn library). Note this is only an example.

The examples/pubsub/server.js runs an HTTP server on port 9000 that accepts WebSocket connections (via websocket.io) that talk the WAMP protocol. For convenience, you can also publish events using HTTP POST. Here the event to be published must be URI encoded in the POST body, like "topicUri=...&body=Hello+World". With this you can implement a veeery "basic" pub sub-hub.

**Comment by Kenzku** [comment]
hmm, it seems I did not understand something - the protocol format [7, "http://example.com/simple", "Hello, world!"] looks different from your implementation format "topicUri=...&body=Hello+World"?

**Comment by nicokaiser** [comment]
My implementation uses the same format [7, "http://example.com/simple", "Hello, world!"], but the protocol for pushing message to the server (not via WAMP but via HTTP POST) is different.
